### ISSUE TITLE
Avoid or improve step by step examples when needed

### ISSUE THREAD

**Issue by davidmanzanares** [CartoDB/carto-vl#1127]
Step by step examples are good. But they are not a silver bullet and I don't think they should be used when the story is not really step by step. For example, the agg guide is structured as follows:
1. Work with `global*`
2. Work with `viewport*`
3. Work with `cluster*`
4. Disable code from steps 1 and 2 since step 3 is not compatible with those previous steps by calling `.off` methods

I think this should be solved in one of the following ways:
- Separate the example in one example with steps 1 and 2 and another stand-alone examples with the code of 3 without the code of steps 1 and 2.
- Make a different map in which step 3 is compatible and produces a map that makes sense in combination with steps 1 and 2

**Comment by VictorVelarde** [comment]
I agree with you. It is a matter of choosing a good style for each guide. Sometimes a step-by-step fits well to "complete a final map" and sometimes it doesn't (and a more "catalogue of examples" approach is needed).

In this particular  case I would try to adapt 'step 3' to better fit the current approach,  so we can avoid removing code and creating off methods.

**Comment by makella** [comment]
related too many maps in zoom based styles hindering map loads : https://github.com/CartoDB/developers/issues/480#issuecomment-442807500

**Comment by VictorVelarde** [comment]
We can decide with @makella if step3 in Aggregation Guide has to be modified, or close this issue
